WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryItems/CoinJoinsHistoryItemViewModel.cs

```diff
@@ -85,6 +85,7 @@ public void Add(TransactionSummary item)
 	private void Refresh()
 	{
 		IsConfirmed = CoinJoinTransactions.All(x => x.IsConfirmed());
+		ConfirmedToolTip = GetConfirmedToolTip(CoinJoinTransactions.Select(x => x.GetConfirmations()).Min());
```

Comment:
This doesn't make much sense, it should only say "Confirmed" since it is a group of individual CJ transactions.

Reply:
So, replacing it by `ConfirmedToolTip = "Confirmed"` will suffice?

Reply:
Yes, but probably in the constructor

Reply:
Then, `IsConfirmed` should be "true" always, and set in the constructor, right?